solarforecastarbiter/metrics/calculator.py

```diff
@@ -251,7 +265,6 @@ def calculate_probabilistic_metrics(processed_fx_obs, categories, metrics,
         datamodel.ProbabilisticForecastConstantValue, and
         the second value are the metrics calculated per
         datamodel.ProbabilisticForecast.
-
```

Comment:
still have a handful of missing empty lines in docstrings in this file

Reply:
I'll check all functions and fix as needed.

Reply:
I think I've fixed all the missing empty lines in the docstrings for all functions modified by this PR.